packages/workspace/src/builders/run-commands/schema.json

```diff
@@ -2,21 +2,32 @@
   "title": "Run Commands",
   "description": "Run any custom commands with Nx",
   "type": "object",
-
   "properties": {
     "commands": {
-      "type": "array",
-      "items": {
-        "type": "object",
-        "properties": {
-          "command": {
-            "type": "string",
-            "description": "Command to run in child process"
+      "oneOf": [
```

Comment:
This currently says `Array<string> | Array<CommandObject>`. Do you intend for `Array<string | CommandObject>`? The code seems to allow for it.